### DISCUSSION TITLE
SharePoint online Manage permission for large list

### DISCUSSION THREAD

**Discussion by krunal039** [pnp/pnpcore#394] [Q&A]
Hi All,

We have large list with around 900000 items across folders and now when we try to manage permission have error with Threshold limit and not allowing us to manage permission.

Is there a way to manage permission for folder/list where threshold has been exceeded ?

**Comment by jansenbe** [comment]
@krunal039 : Can you let us know which specific API call fails...doing a Fiddler trace will help understand where it exactly goes wrong
